README.md

```diff
@@ -35,6 +35,7 @@ ExampleDAOView() => (
 )
 ```
 
+(?)
```

Comment:
This documentation should be changed, my understanding is that we are going to access to method through `data` object